### PR TITLE
Rework evidence security categories

### LINKED ISSUE #15221: Re-categorize evidence security entries
Do a best effort to re-categorize annotated tests under the following categories:

1. Authentication
1. Authorization
1. Confidentiality
1. Integrity
1. Availability
1. Non-Repudiation
1. Resilience

Part of #14254

### PR DESCRIPTION
**Pull request by cbley-da**
using the 7 industry standard security testing areas:

* Authentication
* Authorization
* Confidentiality
* Integrity
* Availability
* Non-Repudiation
* Resilience

Fixes #15221

### CODE REVIEW

File: security/EvidenceSecurity.hs [hunk 1]
```diff
@@ -104,7 +104,14 @@ theParser = some line <* eof
...
 
 
-data Category = Authorization | Privacy | Semantics | Performance | InputValidation | Authentication
+data Category
+  = Authentication
+  | Authorization
+  | Availability
+  | Confidentiality
+  | Integrity
+  -- | NonRepudiation
```
Comment (ray-roestenburg-da) [inline security/EvidenceSecurity.hs:113]: Should these be removed instead of commented?

**Review by ray-roestenburg-da — APPROVED** [review]
Minor comment, otherwise LGTM.

File: security/EvidenceSecurity.hs (hunk 1 above)
Comment (cbley-da) [reply security/EvidenceSecurity.hs]: Fixed.

File: security/EvidenceSecurity.hs [hunk 2]
```diff
@@ -140,9 +147,10 @@ ppDescription Description{filename,lineno,freeText} =
...
-  Privacy -> "Privacy"
-  Semantics -> "Semantics"
-  Performance -> "Performance"
-  InputValidation -> "Input Validation"
   Authentication -> "Authentication"
+  Authorization -> "Authorization"
+  Availability -> "Availability"
+  Confidentiality -> "Confidentiality"
+  Integrity -> "Integrity"
+  -- NonRepudiation -> "Non-Repudiation"
```
[the remark this answers is not in the document]
Comment (cbley-da) [reply security/EvidenceSecurity.hs]: Fixed.

**Pull request merged**